composer.json

```
@@ -39,7 +39,7 @@
     "require-dev": {
         "doctrine/coding-standard": "14.0.0",
         "phpstan/phpstan": "2.1.32",
-        "phpunit/phpunit": "^9.6.13 || 11.4.14",
+        "phpunit/phpunit": "^9.6.13 || 11.4.14 || 12.4.3",
```

```suggestion
        "phpunit/phpunit": "^9.6.13 || ^11.4.14 || ^12.4.3",
```

Maybe there was an issue with some versions, I added a question here:

- https://github.com/doctrine/DoctrineFixturesBundle/pull/499#discussion_r2576645907

---

Ok, i will wait with comitting until i got some feedback.

---

`git log -S 11.4.14` ;)

---

Thanks greg0ire, I didn’t know about this search feature.

It returns:

```
[…]
Date:   Sat Mar 22 18:47:23 2025 +0100

    Pin development tools
    
    We have jobs in our CI that run with unstable deps, but we are not
    really interested in getting unstable deps for our development tools.
```